tests/test_geodesy.py

```diff
@@ -27,7 +27,10 @@ def test_is_within_radius_km():
 
 
 def test_srtm_altitude():
-    assert geodesy.get_topo_altitude(TEST_LAT, TEST_LON) == pytest.approx(207)
+    try:
+        assert geodesy.get_topo_altitude(TEST_LAT, TEST_LON) == pytest.approx(207)
+    except FileNotFoundError:  # this test can sporadically fail
+        pass
```

Comment:
Not sure that this is the way to treat a flaky dependency on CI.

On a local development environment we do not expect this to fail and on a production environment this should not fail.

maybe marking the test with `@pytest.mark.xfail(reason="fails in CI when geonum can not download 'srtm' data", raises=FileNotFoundError)`
as suggested on the [pytest docs](https://docs.pytest.org/en/7.1.x/explanation/flaky.html#xfail-strict)

Reply:
Agreed, this is better. Implemented